magprime/configspec.ini

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
That's fine with me.  After work I'll deploy to staging and edit the ``development.ini`` there and then do the same for production.  Then I can insert last year's season supporters and send out those emails.